docs/upgrade-to-v4.asciidoc

```diff
@@ -59,7 +59,34 @@ via:
 [source,js]
 ----
 const {stringify} = require('querystring');
-console.log( stringify(span.ids, ' ', '=')) );
+console.log(stringify(span.ids, ' ', '='));
 ----
 
 For log correlation with _structured_ logs, see <<log-correlation-ids>>.
+
+[[v4-warnings]]
+==== Log warnings
+
+There are some 
+
+[[v4-warning-duration-units]]
+=====
+
+Configuration options that define a duration like `apiRequestTime` or 
+`exitSpanMinDuration` expect to have their units specified in the value
+(e.g. `10s`, `100ms`). If the unit is not in the value tha agent will
+warn about it and fallback to its default unit which is specified in
+the default value.
+
+[[v4-warning-size-units]]
+=====
+
+Byte size config options like `apiRequestSize` expect to have the size
+units specified in the value (e.g. `10kb`, `1gb`). If the unit is not
+in the value tha agent will warn about it and fallback to `bytes`.
+
+[[v4-warning-error-message-max-length]]
+=====
+
+Agent will warn if the deprecated config option `errorMessageMaxLength`
+is set
```

Comment:
```suggestion
```

I know this looks inconsistent. :)  I'm leaning towards not documenting *all* added new warnings. Only interesting ones.  A warning for a deprecated config option should be self-explanatory.  Sorry for getting you to write more docs.